server/queue.php

```diff
@@ -1,23 +1,33 @@
 <?php
 	$root = realpath($_SERVER["DOCUMENT_ROOT"]);
 	require_once "$root/server/config.php";
+	require_once "$root/server/user.class.php";
+	require_once "$root/server/utilities/dateTimezoneUtilities.php";
 	$conn = $DB_CONN;
 
 	// TODO make this handle multiple locations, if necessary
-	$stmt = $conn->prepare("SELECT DISTINCT appointmentId, scheduledTime, firstName, lastName
+	$stmt = $conn->prepare("SELECT Appointment.appointmentId, scheduledTime,
+		firstName, lastName, timeIn, timeReturnedPapers,
+		timeAppointmentStarted, timeAppointmentEnded, completed, notCompletedDescription
 		FROM Appointment
+		LEFT JOIN ServicedAppointment ON Appointment.appointmentId = ServicedAppointment.appointmentId
 		JOIN Client ON Appointment.clientId = Client.clientId
-		WHERE DATE(Appointment.scheduledTime) = ?
+		WHERE Appointment.scheduledTime >= ? AND Appointment.scheduledTime < ?
 			AND Appointment.archived = FALSE
 		ORDER BY Appointment.scheduledTime ASC");
 
-	$stmt->execute(array($_GET['displayDate']));
+	$timezoneOffset = $_GET['timezoneOffset'];
+	$dates = getUtcDateAdjustedForTimezoneOffset($_GET['displayDate'], $timezoneOffset);
+	$stmt->execute(array($dates['date']->format('Y-m-d H:i:s'), $dates['datePlusOneDay']->format('Y-m-d H:i:s')));
 	$appointments = $stmt->fetchAll(PDO::FETCH_ASSOC);
 
 	// We must only display the first letter of the last name
 	// We do this server-side since we can't disclose the data client-side
-	for ($i = 0; $i < count($appointments); $i++) {
-		$appointments[$i]['lastName'] = substr($appointments[$i]['lastName'], 0, 1);
+	foreach ($appointments as &$appointment) {
+		$appointment['lastName'] = substr($appointment['lastName'], 0, 1);
+		$appointment['scheduledTime'] = getTimezoneDateFromUtc($appointment['scheduledTime'], $timezoneOffset)->format('Y-m-d H:i:s');
+		// TODO FINISH THE OTHERS
```

Comment:
You need to finish the others ones here (if they are not null), take a look at this article about figuring out if a variable is null: https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/